lib/content/gittenify.js

```diff
@@ -0,0 +1,33 @@
+import catNames from 'cat-names';
+import mutationSummary from 'mutation-summary';
+import catPhoto from './catPhoto';
+import replaceUsers from './replaceUsers';
+
+export default function gittenify(store) {
+  const userMap = {};
+
+  for (const u of store.users) {
```

Comment:
Babel transforms this into `for(var u,l=e.users[Symbol.iterator]();!(a=(u=l.next()).done);a=!0)`. I think the variables in the `for`-head only get initialized once, hence why I caught it. I could be wrong though. ES6 sure is interesting ;)